File: hw/chavez/u3/hw23Singleton/Python/tax_calculator.py
from tax import Tax

class TaxCalculator:

    @staticmethod
    def main():
        # Gets the single instance (Singleton)    
        tax = Tax()

        tax.updateTaxPercentage(15.0)

        subtotal = 250.0

        print("Tax Percentage:", str(tax.getPercentage()) + "%")
        print("Subtotal: $", subtotal)
        print("Total with Tax: $", tax.salesTotal(subtotal))


TaxCalculator.main()

File: hw/chavez/u3/hw23Singleton/JavaScript/TaxCalculator.js
const Tax = require("./tax");

class TaxCalculator {

    static main() {

        const tax = Tax.getInstance();
        //console.log(Tax)
        tax.updatetaxPercentage(15.0);

        let subtotal = 250.0;

        console.log("Tax Percentage: " + tax.getPercentage() + "%");
        console.log("Subtotal: $" + subtotal);
        console.log("Total with Tax: $" + tax.salesTotal(subtotal));
    }
}

TaxCalculator.main();